client/src/components/desktop/HomeDesktop.js

```diff
@@ -40,4 +44,43 @@ const HomeBlock2Desktop = () => {
   );
 };
 
-export default HomeBlock2Desktop;
+export const HomeBlock3Desktop = () => {
+  return (
+    <Row className="home-block-3" type="flex" justify="center" align="middle">
+      <Col span={23}>
+        {/* <Carousel autoplay dotPosition="right"> */}
+        <Carousel dotPosition="right">
+          {testimonials.map(element => {
+            return (
+              <div>
```

Comment:
Same thing as above, see if this div is necessary :)

Reply:
idk i removed it and elements repositioned themselves so im gonna leave it there for now